gulpfile.js/lib/task-defaults.js

```diff
@@ -40,7 +40,17 @@ module.exports = {
         "./node_modules"
       ]
     },
-    extensions: ["sass", "scss", "css"]
+    extensions: ["sass", "scss", "css"],
```

Comment:
Thoughts on adding `pcss` as an alternative extension? I don't like using `css` as an extension for files that need processing. If it's not Sass or SCSS, then `pcss` is a nice way of specifying.

Reply:
Great idea 👍